compliance/tags/azure_rg_tags/azure_resource_group_tags.pt

```diff
@@ -36,6 +36,13 @@ parameter "param_tag_key" do
   EOS
 end 
 
+parameter "param_automatic_tag" do
+    type "string"
+    label "Automatically Tag resources"
+    description "When this value is set to yes this policy will automatically take the tag resources action"
```

Comment:
consider changing to:
When this value is set to yes this policy will automatically tag the resources.